portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,62 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link rel="icon" href="images/deeptea-icon.png" type="image/x-icon">
+    <link href='https://fonts.googleapis.com/css?family=Buenard' rel='stylesheet'>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    </head>
+    <body>
+        
+        <div id="profile">
+            <img src="/images/deepti.jpg" alt="Me!" width="300" height="300">
+            <div id="quotes">
+                <p><b>Click here to get a random message:</b></p>
+                <button onclick="addRandomGreeting()">Click me for a Quote!</button>
+                <div id="greeting-container"><i></i></div>
+            </div>
+            
+        </div>
+        <div id="content">
+            <h1><b>Deepti Vaidyanathan</b></h1>
+            <p>I am a 3rd year Computer Science major at the Georgia Institute of Technology 
+                with threads in Intelligence and Information/Internetworks. I am interested 
+                in data science, information security, and artificial intelligence espeicially 
```

Comment:
especially

Reply:
Thanks for catching that! I fixed it in the most recent commit